astroquery/simbad/core.py

```diff
@@ -689,21 +670,18 @@ def query_region_async(self, coordinates, radius=2*u.arcmin,
             if len(set(frame)) > 1:
                 raise ValueError("Coordinates have different frames")
             else:
-                frame = set(frame).pop()
+                frame = frame[0]
 
-            if vector and _has_length(radius) and len(radius) == len(ra):
-                # all good, continue
-                pass
-            elif vector and _has_length(radius) and len(radius) != len(ra):
-                raise ValueError("Mismatch between radii and coordinates")
-            elif vector and not _has_length(radius):
+            if isiterable(radius):
```

Comment:
as I see at this point `radius` can still be a string, this would add a bug for a single string radius for a list of RA case. Maybe also add a test for that case? (as I see vector queries are totally missing from the local tests and for multicord tests only quantity scalar radius is being used for the remote tests)